internal/cli/apps.go

```diff
@@ -298,9 +304,12 @@ func deleteAppCmd(cli *cli) *cobra.Command {
 		Use:   "delete",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Delete an application",
-		Long:  "Delete an application.",
-		Example: `auth0 apps delete 
-auth0 apps delete <id>`,
+		Long: "Delete an application.\n\n" +
+			"To delete interactively, use `auth0 apps delete` with no arguments.\n\n" +
+			"To delete non-interactively, supply the application id and the `--force` " +
+			"flag to skip confirmation.",
+		Example: `  auth0 apps delete 
+  auth0 apps delete <id>`,
```

Comment:
Consider adding the `auth0 apps delete <id> --force` example in a future PR, like it is in other commands.